### PR TITLE
Cache pension funds

### PR DESCRIPTION
**Pull request by ErkoRisthein**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -68,6 +68,7 @@ dependencies {
 	runtimeOnly('org.postgresql:postgresql')
 	implementation("org.springframework.boot:spring-boot-starter-data-jpa")
 	implementation("org.springframework.security.oauth.boot:spring-security-oauth2-autoconfigure:2.1.3.RELEASE")
+	compile('net.sf.ehcache:ehcache')
```
Comment (v3rm0n) [inline build.gradle]: compile deprecated, use implementation

File: src/main/resources/ehcache.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ehcache xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://www.ehcache.org/ehcache.xsd"
+         updateCheck="true" monitoring="autodetect" dynamicConfig="true">
+
+  <cache name="funds"
+         maxElementsInMemory="100"
+         eternal="true"
```
Comment (v3rm0n) [inline src/main/resources/ehcache.xml]: Never update?

Comment (ErkoRisthein) [reply src/main/resources/ehcache.xml]: On restart :trollface:

Comment (ErkoRisthein) [reply src/main/resources/ehcache.xml]: > There are only two hard things in Computer Science: cache invalidation and naming things.
> 
> -- Phil Karlton

**Pull request merged**